lib/chef/resource/apt_repository.rb

```diff
@@ -187,6 +187,22 @@ def extract_fingerprints_from_cmd(*cmd)
           end.compact
         end
 
+        # run the specified command and extract the public key ids
```

Comment:
Is `extract_fingerprints_from_cmd` deprecated by these changes? If so, would recommend removing it.

Reply:
It is still used in `install_key_from_keyserver`. I'm not sure if it could be removed from that though.